geostore/views.py

```diff
@@ -190,6 +190,42 @@ def perform_create(self, serializer):
         layer = self.get_layer()
         serializer.save(layer_id=layer.pk)
 
+    @action(detail=True, methods=['get', 'put', 'patch', 'delete'], permission_classes=[],
```

Comment:
```suggestion
    @action(detail=True, methods=['get', 'put', 'patch', 'delete'],
```
permissions=[] -> ouvert aux 4 vents